be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/RollCallHandler.scala

```diff
@@ -48,12 +51,7 @@ class RollCallHandler(dbRef: => AskableActorRef) extends MessageHandler {
         rollCallChannel: Channel = Channel(s"${Channel.ROOT_CHANNEL_PREFIX}${data.id}")
         laoId: Hash = rpcRequest.extractLaoId
         // do not see how to NOT use the transformWith function except if create other function, i.e. the inverse of CHannelExists
-        _ <- dbActor ? DbActor.ChannelExists(rollCallChannel) transformWith {
-          case Success(_) => Future {
-            throw DbActorNAckException(ErrorCodes.INVALID_ACTION.id, "rollCall already exists in db")
-          }
-          case _ => Future {}
-        }
+        _ <- dbActor ? DbActor.ChannelMissing(rollCallChannel)
```

Comment:
would it make sense to call this `AssertChannelMissing` ?
It might more clearly convey the idea that this WILL fail if our expectation is not met.

Reply:
Good point in my opinion

Reply:
Yes, I agree with this point as well